src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/PrisonAppointment.kt

```diff
@@ -34,7 +34,7 @@ class PrisonAppointment private constructor(
 
   val appointmentType: String,
 
-  @Deprecated(message = "This is to be superseded by notesForPrisoners.")
+  @Deprecated(message = "This is superseded by notesForPrisoners.")
   val comments: String? = null,
```

Comment:
Left on JPA entity for now, could probably be removed (but left on the DB table).